packages/firestore/src/remote/serializer.ts

```diff
@@ -1282,7 +1282,7 @@ export class JsonProtoSerializer {
   }
 
   fromFieldFilter(filter: api.Filter): Filter {
-    return new FieldFilter(
+    return Filter.create(
```

Comment:
There are two other instances of `new FieldFilter()` in this file that I think should optimally be `FieldFilter.create()` as well (though in practice they're not harmful).

To make sure we don't mess this up going forward, I think it might make sense to move `Filter.create()` to be `FieldFilter.create()` so we can make the `FieldFilter` constructor `protected`.  This is probably more correct anyway, since the only kind of filter that Filter.create() could conceivably create with its current arguments is `FieldFilter`.

Reply:
I agree, and I like this suggestion. Done, PTAL.